runtime/runtime.go

```diff
@@ -300,6 +325,18 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 		}
 	}
 
+	if rt.Params.EnableVersionCheck {
+		d := time.Duration(int64(time.Second) * defaultUploadIntervalSec)
```

Comment:
Ideally, there would be some randomization with the interval -- just to simplify the life of the backend.

Reply:
Done